src/modules/mock.command.js

```diff
@@ -0,0 +1,12 @@
+const command = require('./command');
+
+const mock = async function(response, { input, i18n }) {
+	const {user} = response 
+	var lastWord = input.substring(input.lastIndexOf(" ")+1);
+	lastWord = lastWord != '/mock' ? lastWord : 'nothing to mock('
+	var id = user.discordId;
+	response.output = i18n('mock', {id, lastWord});
+    return response;
```

Comment:
good 2 spaces! BUT WHY U USE 4 spaces above?

Reply:
dont understand move to rename discordId to id... use in 18n string discordId instead... and try to write universal modules! Dont lock on client-specific logic/naming on future